src/Command/Site/DbImportCommand.php

```diff
@@ -193,9 +177,45 @@ protected function execute(InputInterface $input, OutputInterface $output) {
   }
 
   /**
-   * Downloads or copies the dump to a local destination.
+   * Helper to return list of commands to import sql dump.
+   */
+  protected function getSqlImportCommands(){
+    $this->io->comment('Importing dump');
+
+    $commands[] = sprintf('cd %s', $this->shellPath($this->getWebRoot()));
+    $commands[] = sprintf('drush sql-create -y');
+    $commands[] = sprintf('drush sql-cli < %s', $this->filename);
```

Comment:
We should, there is another ticket to add progress, but the Symfony progress, not the 'pv' command that is used on our vms which is an external command and we dont want to have this dependency